spec/system/applying/change_in_financial_circumstances_spec.rb

```diff
@@ -0,0 +1,202 @@
+
+      # steps/income/how_manage_with_no_income
+      choose_answer('How does your client manage with no income?', 'They stay with family for free')
+      save_and_continue
+
+      # income cya
+      save_and_continue
+
+      # steps/outgoings/housing_payments_where_lives
+      choose_answer('Which of these payments does your client make where they usually live?',
+                    'They do not make any of these payments')
+      save_and_continue
+
+      # steps/outgoings/pay_council_tax
+      choose_answer('Does your client pay Council Tax where they usually live?', 'No')
+      save_and_continue
+
+      # steps/outgoings/which_payments
+      choose_answers('Which of these payments does your client make?',
+                     ['They do not make any of these payments'])
+      save_and_continue
+
+      # steps/outgoings/client_paid_income_tax_rate
+      choose_answer('Has your client paid the 40% income tax rate in the last 2 years?', 'No')
+      save_and_continue
+
+      # steps/outgoings/are_outgoings_more_than_income
+      choose_answer("Are your client's outgoings more than their income?", 'No')
+      save_and_continue
+
+      # outgoings cya
+      save_and_continue
+
+      # steps/capital/which_assets_owned
+      choose_answer('Which assets does your client own or part-own inside or outside the UK?',
+                    'They do not own any of these assets')
+      save_and_continue
+
+      # steps/capital/which_savings
+      choose_answer('Which savings does your client have inside or outside the UK?',
+                    'They do not have any of these savings')
+      save_and_continue
+
+      # steps/capital/client_any_premium_bonds
+      choose_answer('Does your client have any Premium Bonds?', 'No')
+      save_and_continue
+
+      # steps/capital/any_national_savings_certificates
+      choose_answer('Does your client have any National Savings Certificates?', 'No')
+      save_and_continue
+
+      # steps/capital/which_investments
+      choose_answer('Which investments does your client have inside or outside the UK?',
+                    'They do not have any of these investments')
+      save_and_continue
+
+      # steps/capital/client_benefit_from_trust_fund
+      choose_answer('Does your client stand to benefit from a trust fund?', 'No')
+      save_and_continue
+
+      # steps/capital/income_savings_assets_under_restraint_freezing_order
+      choose_answer(
+        'Does your client have any income, savings or assets under a restraint or freezing order?',
+        'No'
+      )
+      save_and_continue
+
+      # capital cya
+      choose_answers('Confirm the following',
+                     ['Tick to confirm your client does not have any other assets or capital'])
+      save_and_continue
+
+      # steps/evidence/upload
+      # to bypass evidence validation requirements
+      allow_any_instance_of(SupportingEvidence::AnswersValidator).to receive_messages(validate: true,
```

Comment:
I've opted to bypass the evidence upload screen by stubbing the validation check to return `true`. Any objections, let me know!

Reply:
Fine for now, but would be good to bypass the upload part rather than the validation at some point.